projects/route-analyzer/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@vcd/route-analyzer",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "bin": "cli.js",
   "dependencies": {
     "@wessberg/ts-evaluator": "0.0.25",
     "yargs": "15.3.1",
-    "typescript": "3.5.3"
+    "typescript": "3.8.3"
```

Comment:
Shouldn’t typescript be listed as a peer dependency as
```
"peerDependencies": {
   "typescript": "^3.x"
},
```
That way, route-analyzer project uses the TS v3.x from base just as it’s transitive dependency @wessberg/ts-evaluator, which also uses TS as peer dependency from the base project.

Reply:
I was thinking about that also but on the other side route-analyzer may be used as a stand-alone tool, not depending on the fact that it will be used as part of a base project
it is not like a common component library, where the components have to be part of a project and cannot live by themselves alone - at least these were my considerations

Reply:
Angular's guidelines are that libraries should not bundle large packages. They should make them peer dependencies, that's why we are required to add dependencies to a white list. See https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md